tatsu/buffering.py

```diff
@@ -268,11 +268,11 @@ def eat_whitespace(self):
         return self._eat_regex(self.whitespace_re)
 
     def eat_comments(self):
-        comments = self._eat_regex_list(self.config.comments)
+        comments = self._eat_regex_list(self.config.comments_pattern)
```

Comment:
This isn't necessary since you are now doing `cached_re_compile` lookup in `_scanre`, it just means we hit the cache twice: here and in `_scanre`. it should be fine to just reference `self.config.comments` here and get the benefit of the cache